distribution/docs/src/main/resources/content/_architectures/_securityFramework/security-oidc-intro.adoc

```diff
@@ -0,0 +1,82 @@
+:title: Securing REST with OIDC and OAuth 2.0
+:type: subSecurityFramework
+:status: published
+:parent: Web Service Security Architecture
+:order: 01
+:summary: Securing REST with OIDC and OAuth 2.0
+
+==== {title}
+
+To use https://openid.net/specs/openid-connect-core-1_0.html[OpenID Connect (OIDC)] and https://tools.ietf.org/html/rfc6749[OAuth 2.0] to secure REST endpoints,
+${branding} needs to be connected to an external Identity Provider (IdP) which supports these protocols.
+
+===== OIDC
+
+OIDC is used to authenticate (or log in) a user. To use this protocol in ${branding}, ${branding} needs the external IdP's information.
+
+To connect to an external IdP,
```

Comment:
To get this list to format correctly you can either: 
- add a blank space between this line and the list
or 
- Format this line as a caption, such as `.Connecting to an External IdP`